test/functional/styles/icons.js

```diff
@@ -0,0 +1,90 @@
+const variants = [
+  'external',
+  'arrow-up',
+  'arrow-down',
+  'feedback',
+  'spinner',
+  'search',
+  'language',
+  'facebook',
+  'twitter',
+  'googleplus',
+  'linkedin',
+  'rss',
+  'up',
+  'right',
+  'down',
+  'left',
+  'file',
+  'audio',
+  'brochure',
+  'edit',
+  'image',
+  'download',
+  'infographic',
+  'multiple-files',
+  'organigram',
+  'package',
+  'slides',
+  'video',
+  'error',
+  'success',
+  'warning',
+  'check',
+  'generic-lang',
+  'budget',
+  'digital',
+  'energy',
+  'euro',
+  'global',
+  'growth',
+  'regulation',
+  'tag-close',
+  'tag-close-hover',
+  'calendar',
+  'location',
+  'livestreaming',
+  'book',
+  'data',
+  'copy',
+  'spreadsheet',
+  'presentation',
+  'info',
+  'share',
+  'faq',
+  'camera',
+  'user',
+  'logout',
+];
+
+variants.forEach(variant => {
```

Comment:
I think it'd be better to test `components/preview/ecl-icons` which has them all. That covers basically all modifiers and variants at the same time, 1 image per browser.